src/main/kotlin/org/hyacinthbots/lilybot/extensions/config/Config.kt

```diff
@@ -242,20 +252,25 @@ suspend fun Config.configCommand() = unsafeSlashCommand {
 				}
 			}
 
-			checkChannel(guild, interactionResponse)?.createMessage {
-				embed {
-					moderationEmbed()
-				}
-			}
-
 			ModerationConfigCollection().setConfig(
 				ModerationConfigData(
 					guild!!.id,
 					arguments.enabled,
 					arguments.modActionLog?.id,
-					arguments.moderatorRole?.id
+					arguments.moderatorRole?.id,
+					arguments.logPublicly
 				)
 			)
+
+			checkChannel(
```

Comment:
This should happen before we set the config so if something error's we're not putting bad values in the DB

Reply:
The way the check works it needs to be done after the fact, though we can do a manual check on the channel before. I'll write that now